netbox/extras/reports.py

```diff
@@ -85,9 +85,9 @@ class Report(object):
     description = None
     scheduling_enabled = True
     job_timeout = None
+    test_methods = []
```

Comment:
This list won't be populated if the report class hasn't been initialized. IMO it would be preferable to omit the attribute entirely rather than to assert that there are no test methods.